core/src/test/resources/server.conf

```diff
@@ -47,6 +47,8 @@ http.query.cache.enabled=false
 http.query.cache.block.count=32
 http.query.cache.row.count=16
 
+http.pessimistic.health.check.enabled=true
```

Comment:
do we have to use a different default here?

Reply:
This is a test-only file, so I'm not sure if I understand the question.

Reply:
I am soryr, I should have been more explicit. 

My assumption: Properties from this file are used in tests which use the `PropServerConfiguration`. 
A question: Is it a good idea these tests to have a default which differs from production server.conf? What's the benefit of that?

Reply:
We have both `testAllDefaults` and `testSetAllFromFile` tests in `PropServerConfigurationTest`. The first one doesn't use the test `server.conf` file while the second one does. The purpose of `testSetAllFromFile` is to make sure that we correctly read non-default prop values from the file.